### ISSUE TITLE
Shape Notes not appearing in Plugins menu

### ISSUE THREAD

**Issue by edpw** [lasconic/shape_notes#2]
MuseScore 2.0

1. I put the Shape Note plugin files shape_notes.js shape_notes.qml and shape_notes.ui in /usr/share/mscore-2.0/plugins
2. The shape_notes plugin appears in the Plugin Manager. I select it.
3. The Shape Notes plugin does *not* then appear in the Plugins menu.

The Shape Notes plugin should appear in the Plugins menu so I can convert round notes to shape notes.

**Comment by lasconic** [comment]
On which linux distrib?

**Comment by edpw** [comment]
Ubuntu 14.10

**Comment by lasconic** [comment]
Are other plugin working?

**Comment by edpw** [comment]
Yes. I've tried a few other plugins and they've all worked, for example Note Names, Colour Notes, Create Score, Scoreview.

Thanks for your help with this by the way.

**Comment by edpw** [comment]
Hey, just wondering whether you've had time to get anywhere this this?

**Comment by edpw** [comment]
I've managed to fix this. I'm using Ubuntu 15.04 64 bit. Musescore needs this package for the Shapenote and a few of the default plugins to run: qml-module-qtquick-controls

Run this command in a Terminal
sudo apt-get install qml-module-qtquick-controls

**Comment by lasconic** [comment]
So it's a packaging issue in your distrib. Where did you get the MuseScore package?
Is it the same issue? https://bugs.launchpad.net/ubuntu/+source/musescore/+bug/1444318
If yes, please add your voice.
